src/com/facebook/buck/apple/project_generator/ProjectGenerator.java

```diff
@@ -2716,17 +2716,14 @@ ProductType bundleToTargetProductType(
           if (productType.isPresent()) {
             return productType.get();
           }
-        } else {
-          switch (extension) {
-            case FRAMEWORK:
-              return ProductType.STATIC_FRAMEWORK;
-          }
+        } else if (extension == FRAMEWORK) {
```

Comment:
This makes the `@SuppressWarnings("incomplete-switch")` of the method unnecessary, and `FRAMEWORK` needs to be be `AppleBundleExtension.FRAMEWORK`.